### PR TITLE
Handle dynamic css imports that are also statically imported

### PR DESCRIPTION
**Pull request by benmccann**
Fixes https://github.com/sveltejs/sapper/issues/1493. See the issue for a description of the change

### CODE REVIEW

**Comment by Conduitry** [comment]
Can we add a test for this as well? These were obviously weird edge cases that were never considered, and it would be nice to have some ability to guard against them going forward.

File: src/core/create_compilers/RollupCompiler.ts [hunk 1]
```diff
@@ -167,37 +169,45 @@ export default class RollupCompiler {
...
 					}
+				}
 
-					// Put everything that's leftover into the entry chunk in order to include css
-					// that is in a dynamically imported chunk
-					const chunks_for_tree = js_deps(tree_entry_chunk,
-						{ walk: ctx => !subtree || ctx.chunk.fileName !== entry_chunk.fileName });
-					const unused = new Set(chunks_for_tree.filter(x => !processed_chunks.has(x)));
-					handle_chunks(tree_entry_chunk, unused);
+				function isRoute(file_path: string) {
```
Comment (Conduitry) [inline src/core/create_compilers/RollupCompiler.ts:180]: `is_route` please, to keep with the snake case used elsewhere.

Comment (benmccann) [reply src/core/create_compilers/RollupCompiler.ts]: done

**Comment by benmccann** [comment]
Ok, I updated the tests so that they fail on `master` and pass with this PR. I didn't need to write a new test but just added a new route that would cause existing tests to fail

**Comment by Conduitry** [comment]
Oh nice, I see that this also makes `rollup-dependency-tree` a dev/bundled dependency.

**Comment by benmccann** [comment]
My reason for moving it was that it's only used in building the application by Rollup / the Sapper build process and so I was thinking it was a dev dependency. E.g. if someone runs `npm ci --prod` then I figured this would stop it from being a dependency that's shipped to production. It works just fine either place I have it and I don't have much of an opinion on where to put it. Were you thinking I should move it back?

**Comment by Conduitry** [comment]
No no, it's nice having it be bundled in instead. My understanding from the conversation [here](https://github.com/sveltejs/sapper/pull/1452#discussion_r479437894) was that you were running into some issues when doing that, but it didn't seem very worthwhile to try to get to the bottom of it, because Sapper itself is only a dev dependency of apps that are written in it, and so built apps didn't have any additional dependencies.

I'm a bit confused by parts of your last comment. I would expect that the only difference for someone installing `sapper` into their app would be whether they get a bundled copy of `rollup-dependency-tree` or their package manager has to separately request and install it.

**Comment by benmccann** [comment]
I think we may have been talking about different things when you said "external" in https://github.com/sveltejs/sapper/pull/1452#discussion_r479437894. I originally thought you meant was there a reason to make it a library and get it from npm instead of just putting the source code in Sapper. But now I understand you mean `devDependencies` vs not, so I was answering a bit of a different question on the other thread.

Feel free to ignore my last comment too :smile: I get a bit confused sometimes about Sapper being both a builder and a runtime in the same project

Anyway, I don't think there's much difference whether this is in `devDependencies` or `dependencies`, so I'm happy to have it in either place
